### PR TITLE
Update README to include latest changes

### PR DESCRIPTION
**Pull request by redbaty**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,7 +118,12 @@ public class PresidentClassMap : ExcelClassMap<President>
 using (var stream = File.OpenRead("Presidents.xlsx"))
 using (var importer = new ExcelImporter(stream))
 {
+    //You can register class maps by type
     importer.Configuration.RegisterClassMap<PresidentClassMap>();
+    
+    //Or by namespace
+    importer.RegisterMapperClassesByNamespace("NAMESPACE");
```
Comment (hughbe) [inline README.md]: Maybe rename this to be ‘My.Namespace`?

File: README.md [hunk 2]
```diff
@@ -118,7 +118,12 @@ public class PresidentClassMap : ExcelClassMap<President>
 using (var stream = File.OpenRead("Presidents.xlsx"))
 using (var importer = new ExcelImporter(stream))
 {
+    //You can register class maps by type
```
Comment (hughbe) [inline README.md]: Nit: space between slashes and start of sentence and a full stop at the end

File: README.md (hunk 1 above)
Comment (redbaty) [reply README.md]: Done, do you think it's a good idea to leave My.Namespace without quotes ? So that if someone copy-pastes it he'll immediately see.

**Comment by hughbe** [comment]
Thanks!

**Pull request merged**